File: app/scripts/controllers/downloads.js
(function(){
    var nmsReportsApp = angular
        .module('nmsReports')
        .controller("DownloadsController", ['$scope', '$state', '$http', '$sce', function($scope, $state, $http, $sce){

            $http.get(backend_root + 'page/downloads')
                .then(function(result){
                        if(result.status===200){
                            $scope.downloadsPage= result.data.pagecontent;
                            $scope.downloadsPageContent = $sce.trustAsHtml($scope.downloadsPage);
                        }
                        else {
                            $state.go('login', {});
                        }
                    }, function(error){
                        $state.go('login', {});
                    }
                )


            $scope.downloadList = [
                 {"id":1,"name":"downloads1"},
                 {"id":2,"name":"downloads2"},
                 {"id":3,"name":"downloads3"},
                 {"id":4,"name":"downloads4"},
            ];

        }])
})()

File: NMSReportingSuite/src/main/java/com/beehyv/nmsreporting/htmlpages/Downloads.java
package com.beehyv.nmsreporting.htmlpages;

public class Downloads {
    public static String pageContent ="<div data-ng-app=\"nmsReports\" class=\"\"  data-ng-controller=\"DownloadsController\">" +
            "    <div>Downloads Page</div>\n" +
            "    <ul>\n" +
            "        <li data-ng-repeat=\" downloaditem in downloadList\">\n" +
            "           {{downloaditem.id}} : {{downloaditem.name}}\n" +
            "        </li>\n" +
            "\n" +
            "    </ul>" +
            "</div>";
}
